server/views/pages/activities/suspensions/confirmation.njk

```diff
@@ -15,8 +15,22 @@
                         {{ session.suspendJourney.inmate.prisonerName | safe }} ({{ session.suspendJourney.inmate.prisonerNumber }}) is now suspended from
                         {% if session.suspendJourney.allocations | length == 1 %}
                             {{ session.suspendJourney.allocations[0].activityName | safe }}
+                            {% if suspendPrisonerWithPayToggleEnabled %} 
+                                {% if session.suspendJourney.paid == 'YES' %} 
```

Comment:
It would be good to use the enum here instead of a string e.g.
session.suspendJourney.paid == YesNo.YES